src/CancelRequestDialog.js

```diff
@@ -129,29 +144,33 @@ class CancelRequestDialog extends React.Component {
           onChange={this.onChangeNotify}
         />
         */}
-        <TextArea
-          label={formatMessage(
-            { id: 'ui-requests.cancel.additionalInfoLabel' },
-            { required: reason.requiresAdditionalInformation ? '*' : ' ' }
+        <FormattedMessage id={additionalInfoPlaceholder}>
+          {message => (
```

Comment:
In such constructs pleaser use meaningful names. In current case it is not clear what is message.
lease rename it to placeholder